usecases/modules/modules.go

```diff
@@ -171,6 +171,21 @@ func (m *Provider) ExtractParams(arguments map[string]interface{}) map[string]in
 	return exractedParams
 }
 
+// ValidateParam validates module parameters
+func (m *Provider) ValidateParam(name string, value interface{}) error {
```

Comment:
Should we give this method (and also the one below - `VectorFromParams`) a slightly more descriptive name. I.e. I think it's not immediately obvious that param refers to a GraphQL params instead of say a schema configuration param.

Reply:
maybe `ValidateNearParam`, `ValidateSearchParameter`, `ValidateSearchParam` and then we would have `VectorFromSearchParams`

Reply:
I like it! Maybe just consistently use `Param` (the second right now has `Parameter`) :)

Reply:
so I'm changing it to `ValidateSearchParam` and `VectorFromSearchParam`

Reply:
I have also changed `ExtractParams` -> `ExtractSearchParams`